dali/pipeline/workspace/workspace.h

```diff
@@ -50,25 +51,25 @@ class ArgumentWorkspace {
     argument_inputs_.clear();
   }
 
-  void AddArgumentInput(shared_ptr<TensorList<CPUBackend>> input, const std::string &arg_name) {
+  void AddArgumentInput(shared_ptr<TensorVector<CPUBackend>> input, const std::string &arg_name) {
```

Comment:
Switch the order of arguments. It should be `key, value`, not `value, key` as it is now.

Reply:
done